.travis.yml

```
@@ -14,6 +14,10 @@ services:
 
 jobs:
   include:
+    - stage: PythonLint
+      os: linux
+      script: "[[ -z \"$(git ls-files | xargs egrep -l '\s+$')\" ]]"
```

I think the backslash needs escaping for yaml:
```suggestion
      script: "[[ -z \"$(git ls-files | xargs egrep -l '\\s+$')\" ]]"
```

---

Yeah, that's all it needed. Travis failed with master sources and passed with fixed sources.